app/Models/Announcement.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Announcement extends Model
+{
+    use HasFactory;
+
+    protected $fillable = [
+        'name',
+        'description',
+        'status',
+        'start_time',
+        'end_time',
+        'user_id',
+    ];
+
+    
+    /**
+     * Announcement belongs to the user
```

Comment:
/**
     * Get the owner of the announcement.
     *
     * @return \Illuminate\Database\Eloquent\Relations\BelongsTo
     */